test/SILOptimizer/lifetime_dependence/semantics.swift

```diff
@@ -417,7 +417,7 @@ public func testTrivialInoutBorrow(p: inout UnsafePointer<Int>) -> Span<Int> {
 
 private let immortalInt = 0
 
-private let immortalString = ""
+private let immortalStrings: [String] = []
```

Comment:
If `String` is marked addressable, escape diagnostics behave slightly differently when it is a global. This seems incidental and could be addressed separately if it is an issue. Changing the type of this variable to get the expected diagnostics.